src/events.psm1

```diff
@@ -739,7 +739,7 @@ function Edit-SafeguardEventSubscription
         if ($PSBoundParameters.ContainsKey("UserToSubscribe"))
         {
             $local:UserEmailAddress = (Get-SafeguardUser -AccessToken $AccessToken -Appliance $Appliance -Insecure:$Insecure -UserToGet $UserToSubscribe).EmailAddress
-            If([string]::IsNullOrWhitespace($local:User.EmailAddress))
```

Comment:
@Goelrachit 
Did you mean to use the variable from the line above here?  I'm proposing that you did.

Reply:
The intention here is same as in New-SafeguardEventSubscription but the check for email address should be a bit different as a UserId might already exist in the exiting subscription and the intent might be to just change the subscription type to email. In that case, the user (running the cmdlet) need not provide a new UserId. So, I need to check for UserId against $local:Body.UserId. I will test and make the necessary change/